Uchu.StandardScripts/NinjagoMonastery/SkeletonTowers.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Uchu.Core;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.NinjagoMonastery
+{
+    [ZoneSpecific(2000)]
+    public class SkeletonTowers : NativeScript
+    {
+        public override Task LoadAsync()
+        {
+            var towerSpawnerObjects = Zone.GameObjects.Where(gameObject =>
+                gameObject.TryGetComponent<SpawnerComponent>(out var spawnerComponent)
+                && spawnerComponent.SpawnTemplate == 15953);
+
+            foreach (var towerSpawnerObject in towerSpawnerObjects)
+            {
+                SetUpLegs(towerSpawnerObject);
+
+                Listen(towerSpawnerObject.GetComponent<SpawnerComponent>().OnRespawnTimeCompleted, p =>
+                {
+                    // Recreate leg spawners, they were destroyed when the tower was smashed
+                    SetUpLegs(towerSpawnerObject);
+                });
+            }
+
+            return Task.CompletedTask;
+        }
+
+        private void SetUpLegs(GameObject towerSpawnerObject)
+        {
+            var towerSpawnerComponent = towerSpawnerObject.GetComponent<SpawnerComponent>();
+            var legSpawnerComponents = new List<SpawnerComponent>();
```

Comment:
Storing the game objects and then getting their components is a neater solution imo